### ISSUE TITLE
Redacting an event should redact all of the reactions

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#9843]
This probably needs server-side support to avoid making potentially thousands of calls to /redact

**Comment by lampholder** [comment]
Is this necessarily true?

If I say something inflamatory and everybody boos, is it right that I can rewrite history by redacting the inflamatory statement?

I think I would prefer that the record at least show that `@bob:example.com` said _something_ and got 9000 👎s (of course all of the booers would be welcome to redact their boos if they chose)

**Comment by manuroe** [comment]
Kamino cloned this issue to [vector-im/riot-ios](https://github.com/vector-im/riot-ios/issues/2546)

**Comment by manuroe** [comment]
Kamino cloned this issue to [vector-im/riotX-android](https://github.com/vector-im/riotX-android/issues/286)

**Comment by turt2live** [comment]
I don't think this needs product consideration. If the user has permission to redact an event, all the reactions should also be redacted.

From a spec side we can do more to actually support this, but a client-side solution would be suitable for now.

**Comment by turt2live** [comment]
sorry, for clarity: if the user can *also* redact the reactions. Moderators/admins might want to clear the message preview for those with reactions enabled.
